CMakeLists.txt

```
@@ -35,6 +35,7 @@ endif()
 option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
 option(SOLC_STATIC_STDLIBS "Link solc against static versions of libgcc and libstdc++ on supported platforms" OFF)
 option(STRICT_Z3_VERSION "Use the latest version of Z3" ON)
+option(DISABLE_WARNINGS "Disable all warning flags, treating warnings as errors and all other pedantic build flags" OFF) 
```

You have some trailing whitespace here:
```suggestion
option(DISABLE_WARNINGS "Disable all warning flags, treating warnings as errors and all other pedantic build flags" OFF)
```

That's why the `chk_coding_style` CI job is failing.